matrix/mat64/lq.go

```diff
@@ -58,10 +58,15 @@ func (lq *LQ) Factorize(a Matrix) {
 // TODO(btracey): Add in the "Reduced" forms for extracting the m×m orthogonal
 // and upper triangular matrices.
 
-// LFromLQ extracts the m×n lower trapezoidal matrix from a LQ decomposition.
-func (m *Dense) LFromLQ(lq *LQ) {
+// LTo extracts the m×n lower trapezoidal matrix from a LQ decomposition.
```

Comment:
Do we need the `To` on the function calls? It's pretty clear from the function signature alone that it produces the matrix `L`, and very clear with the documentation. Seeing `x := lq.L(nil)` or `lq.L(x)` both feel well in line with our other signatures.